hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmMultipartKeyInfo.java

```diff
@@ -293,18 +293,18 @@ public Builder setUpdateID(long id) {
       return this;
     }
 
-    @Override
-    public Builder withUpdateID(long newValue) {
-      super.withUpdateID(newValue);
-      return this;
-    }
-
     public Builder setParentID(long parentObjId) {
       this.parentID = parentObjId;
       return this;
     }
 
     public OmMultipartKeyInfo build() {
+      validate();
```

Comment:
Instead of each build method calling validate we can call super.build() and that could call validate() method. The changes wouldn't be this big.